src/ucs/sys/sock.c

```diff
@@ -37,6 +37,17 @@ typedef ssize_t (*ucs_socket_iov_func_t)(int fd, const struct msghdr *msg,
                                          int flags);
 
 
+static void ucs_socket_print_error_info(int sys_errno)
```

Comment:
seems parameter is always `errno`, maybe not needed?

Reply:
I just wanted to make sure that no one changes this value
other wrapper does this as well

Reply:
ok